packages/bpk-component-banner-alert/readme.md

```diff
@@ -34,5 +34,9 @@ export default () => (
 | message           | node                 | true     | -             |
 | ariaLive          | ARIA_LIVE (one of)   | false    | 'assertive'   |
 | children          | node                 | false    | null          |
-| toggleButtonLabel | string               | false    | null          |
 | className         | string               | no       | null          |
+| dismissable       | bool                 | false    | false         |
```

Comment:
This `dismissable` prop is a little problematic i.e. what happens if consumers provide `children` and `dismissable` - does the toggle button also show?

Perhaps they should be different components - I'm not sure what the best approach is here.

Reply:
I think we also need to contemplate the API here (also in terms of aligning it to the RN component when we do the work).

Can we provide consumers with an `onDismissed` and a `dismissed` prop and allow them to handle the show/hide state

Reply:
Have a look at some of the examples out in the wild https://js.coach/react/?search=alert

A common approach seems to be to have a container component which renders nothing by default. Consumers add / remove alerts to the container by calling a static `add` function (an example exists in https://js.coach/react/react-alert?search=alert).